lib/nightmare.js

```diff
@@ -91,6 +91,22 @@ function Nightmare(options) {
     debug('electron child process exited with code ' + code + ': ' + help[code]);
   });
 
+  Object.defineProperty(this, 'engineVersions', {
```

Comment:
Wouldn’t this make more sense of the Nightmare constructor rather than on an instance? (Same for `.version`)